src/openforms/formio/typing.py

```diff
@@ -49,3 +49,5 @@ class Component(TypedDict):
     hidden: bool
     defaultValue: JSONValue
     prefill: PrefillConfiguration
+    errors: JSONObject | None
+    translatedErrors: JSONObject | None
```

Comment:
this will cause many type "errors" in other places because there's no way (yet, Python 3.11 has it) to specify that keys are optional. So any existing tests without these component keys will  show type errors in IDE's/editors.